rules/windows/process_creation/proc_creation_win_purplesharp_indicators.yml

```diff
@@ -6,21 +6,21 @@ references:
     - https://github.com/mvelazc0/PurpleSharp
 author: Florian Roth
 date: 2021/06/18
-modified: 2022/01/12
+modified: 2023/01/31
 tags:
     - attack.t1587
     - attack.resource_development
 logsource:
     category: process_creation
     product: windows
 detection:
-    selection1:
-        CommandLine|contains:
-            - xyz123456.exe
-            - PurpleSharp
-    selection2:
-        OriginalFileName: 'PurpleSharp.exe'
-    condition: selection1 or selection2
+    selection:
+        - Image|contains: '\purplesharp'
+        - OriginalFileName: 'PurpleSharp.exe'
+        - CommandLine|contains:
+            - 'xyz123456.exe'
+            - 'PurpleSharp'
+    condition: selection
```

Comment:
```
    selection_image:
        - Image|contains: '\purplesharp'
        - OriginalFileName: 'PurpleSharp.exe'
    selection_cmd:
       CommandLine|contains:
            - 'xyz123456.exe'
            - 'PurpleSharp'
    condition: 1 of selection_*
```
It is the same but it is easier to read (  2 ways to find the threat)

Reply:
I did that in https://github.com/SigmaHQ/sigma/pull/3990/commits/2f6d1f042cfb5e92b0f9546cb9a5444a3e55d8de and thought maybe we combine XD.

I reverted the changes. 😛